### PR TITLE
Add lemonade and taco seasoning recipes

### PR DESCRIPTION
**Pull request by ifrendas**

### CODE REVIEW

File: src/taco-seasoning.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- 1/2 tsp seasoned salt
+- 1/2 tsp paprika
+- 1/2 tsp garlic salt
+- 1/4 tsp cumin
+- 1/4 tsp cayenne pepper
+- 1/4 tsp sugar
+
+## Directions
+
+1. Mix ingredients in a bowl
```
Comment (DGKSK8LIFE) [inline src/taco-seasoning.md:19]: These directions are incomplete.

**Review by DGKSK8LIFE — CHANGES_REQUESTED** [review]

File: src/taco-seasoning.md (hunk 1 above)
Comment (ifrendas) [reply src/taco-seasoning.md:19]: How? Once you mix the ingredients the seasoning is made. It's tagged as basic because it isn't stand alone.

Comment (DGKSK8LIFE) [reply src/taco-seasoning.md:19]: Well if that's the only direction, could you elaborate about mixing ingredients into the bowl? Like are you using any specific utensil or?

Comment (ifrendas) [reply src/taco-seasoning.md]: I changed the wording in the latest commit to hopefully make it more clear.

**Review by DGKSK8LIFE — APPROVED** [review]

File: src/lemonade.md [hunk 2]
```diff
@@ -8,9 +8,9 @@ A simple lemonade recipe
 
 ## Ingredients
 
-- 1.75 cups white sugar
+- 1¾ cups white sugar
 - 8 cups water
-- 1.5 cups lemon juice
+- 1½ cups lemon juice
```
Comment (DGKSK8LIFE) [inline src/lemonade.md]: Imo the previous formatting was better.

**Review by DGKSK8LIFE — CHANGES_REQUESTED** [review]

File: src/lemonade.md (hunk 2 above)
Comment (ifrendas) [reply src/lemonade.md]: Ah, my bad. I'll change it back.

**Review by DGKSK8LIFE — APPROVED** [review]

**Pull request closed without merging**